cmd/soroban-rpc/lib/preflight/src/preflight.rs

```diff
@@ -52,6 +52,8 @@ pub(crate) fn preflight_invoke_hf_op(
         .context("cannot set debug diagnostic level")?;
     host.set_ledger_info(ledger_info.clone())
         .context("cannot set ledger info")?;
+    host.set_base_prng_seed([0; 32])
+        .context("cannot set base prng seed")?;
```

Comment:
@tsachiherman I think we should probably use a random value here so as to better represent the changing value that will occur on chain. Wdyt?

Reply:
I think that it would be good idea to use a random number for pubnet/testnet/futurenet and a zero for standalone.
But regardless of that decision, I think that this fix, as is - is a must.